torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -226,13 +226,31 @@ void GpuLower::lower(Fusion* fusion, DataType index_type) {
 
   FusionGuard fg(fusion_);
   // prepare for lowering
+  if (isDebugDumpEnabled(DebugDumpOption::PassTransform)) {
+    std::cout << "Before validateIr:" << std::endl;
+    for (auto exp : fusion_->exprs()) {
+      std::cout << exp->toString() << std::endl;
+    }
+  }
```

Comment:
Let's create a function to do this so that we don't need to have this code repeatedly. 

```
void dumpExprs(const std::vector<Expr*>& exprs, std::string msg_pre) {
 if (isDebugDumpEnabled(DebugDumpOption::PassTransform)) {
    std::cout << msg_pre << ":" << std::endl;
    for (auto exp : exprs) {
      std::cout << exp->toString() << std::endl;
    }
  }
}
```

Ideally, we should have an actual pass manager to do this kind of pre- and post-processing automatically, which should also automatically enforce correct dependencies between passes, but for now, I think this should be fine.

Reply:
done. move to dumpExprsIfEnabled(const std::vector<Expr*>& exprs, std::string msg_pre)